### PR TITLE
Add Homestead

### PR DESCRIPTION
**Pull request by kennith**
Valet is only available for macOS. Adding Homestead for Windows developer.

### CODE REVIEW

File: learn.php [hunk 1]
```diff
@@ -30,12 +30,18 @@
         'name' => 'Local PHP Environment',
         'children' => [
             [
-                'name' => 'Valet',
+                'name' => 'Valet (macOS)',
```
Comment (mattstauffer) [inline learn.php]:
```suggestion
                'name' => 'Valet',
```

File: learn.php [hunk 2]
```diff
@@ -30,12 +30,18 @@
...
                 'links' => [
                     'Laravel Docs' => 'https://laravel.com/docs/valet',
                     'Laracasts Video' => 'https://laracasts.com/series/setup-a-mac-dev-machine-from-scratch/episodes/10',
                 ],
             ],
+            [
+                'name' => 'Homestead (Windows or macOS)',
```
Comment (mattstauffer) [inline learn.php]:
```suggestion
                'name' => 'Homestead',
```

**Comment by mattstauffer** [comment]
@kennith I made a few suggestions, which you marked as resolved without merging. Did you intend to merge them? Or dismiss them? Thanks!

**Comment by kennith** [comment]
@mattstauffer Yes, I agree your suggestion and I intend to merge them.

**Comment by mattstauffer** [comment]
FWIW, this introduces the idea that some of these are "either or" type scenarios. Not entirely sure how to handle that in our current stack. I wouldn't want anyone to touch homestead during the learning phase who didn't have to.

**Pull request merged**